core-web/libs/data-access/src/lib/dot-site/dot-site.service.ts

```diff
@@ -42,21 +41,21 @@ export class DotSiteService {
      * @memberof DotSiteService
      */
     getSites(filter = '*', perPage?: number): Observable<Site[]> {
-        return this.http
-            .get<{ entity: Site[] }>(this.siteURL(filter, perPage))
+        return this.#http
```

Comment:
We usually use URLSearchParams for handle the params.
```
const searchParams = new URLSearchParams({
            filter,
            per_page: perPage || PER_PAGE,
            ...this.defaultParams
        });

return this.http.get<{ entity: Site[] }>(`
          ${BASE_SITE_URL}?${searchParams.toString()}
          `).pipe(pluck('entity'));
```

Reply:
@oidacra done!